cli.go

```diff
@@ -40,6 +40,12 @@ func NewCLI() *cli.App {
 				Aliases: []string{"R"},
 				EnvVars: []string{"CYCLIST_REDIS_URL", "REDIS_URL"},
 			},
+			&cli.UintFlag{
```

Comment:
I'm not sure. I don't feel strongly about it, it mostly just surprised me. The usage says that the unit is seconds, so it _is_ documented… I think I'm +0 too.